crates/cibse_guide_e/src/chapter_10/equation_10_11.rs

```diff
@@ -0,0 +1,23 @@
+pub mod openfire_runner;
+
+pub fn limiting_velocity(q: f64, z: f64) -> f64 {
+    return 0.057 * (q / z).powf(1.0/3.0);
+}
+
+pub fn limiting_velocity_symbols(v_e: String, q: String, z: String) -> String {
```

Comment:
```suggestion
pub fn equation(v_e: String, q: String, z: String) -> String {
```

Same suggestion on naming